backend/sound/Mock_MusicMachine.py

```diff
@@ -0,0 +1,43 @@
+import pickle
+from pythonosc import dispatcher, osc_server
+from UDPClient import Client_MusicServer
+
+def rules(cat):
+    if cat == "Kritik":
+        return 0
+    elif cat == "Lob":
+        return 1
+
+class Live:
+    def __init__(self, client, message):
+        self.client = client
+        self.message = message
+        self.song_position = 0
+
+    def message_handler(self, address, map):
+        map = pickle.loads(map)
```

Comment:
python has a builtin function called `map`. It's a bit dangerous to name variables with name that have a different meaning in a different context, so its better to use a different name here (PyStorm should also show a warning like `Shadows built-in  name map`).